<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupReportMonthActionsDialog</class>
 <widget class="QDialog" name="SetupReportMonthActionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="10" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Месяц</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbFinance">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkFinance">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkClass">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Класс</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkActionTypeGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Группа</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Статус</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkOrgStructure">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="chkPerson">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Исполнитель</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="chkAssistant">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ассистент</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="cmbClass">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Статус</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Диагностика</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Лечение</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Прочие мероприятия</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Назначено</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="CActionTypeComboBox" name="cmbActionTypeGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="cmbStatus">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbPerson">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbAssistant">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateEdit" name="edtDate">
     <property name="displayFormat">
      <string>MM.yyyy</string>
     </property>
     <property name="currentSectionIndex">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="chkTissueType">
     <property name="text">
      <string>Тип биоматериала</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbTissueType">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CActionTypeComboBox</class>
   <extends>QComboBox</extends>
   <header>Events/ActionTypeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtDate</tabstop>
  <tabstop>chkFinance</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>chkClass</tabstop>
  <tabstop>cmbClass</tabstop>
  <tabstop>chkActionTypeGroup</tabstop>
  <tabstop>cmbActionTypeGroup</tabstop>
  <tabstop>chkStatus</tabstop>
  <tabstop>cmbStatus</tabstop>
  <tabstop>chkOrgStructure</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>chkPerson</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>chkAssistant</tabstop>
  <tabstop>cmbAssistant</tabstop>
  <tabstop>chkTissueType</tabstop>
  <tabstop>cmbTissueType</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetupReportMonthActionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupReportMonthActionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFinance</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbFinance</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkClass</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbClass</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkActionTypeGroup</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbActionTypeGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkStatus</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbStatus</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOrgStructure</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbOrgStructure</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPerson</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbPerson</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAssistant</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbAssistant</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkTissueType</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbTissueType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
